src/Core/CoreKernel.php

```diff
@@ -357,28 +357,47 @@ protected function getDatabaseConfig()
     }
 
     /**
+     * @return string
+     */
+    protected function getDatabasePrefix()
+    {
+        $prefix = getenv('SS_DATABASE_PREFIX');
+
+        if ($prefix === false) {
+            return '';
```

Comment:
if prefix is false (not declared) shouldn't it be `SS_`? Why does null map to `SS_` but not specifying it maps to empty string?